grammars/go.cson

```diff
@@ -105,11 +105,19 @@
   }
   {
     'name': 'keyword.go'
-    'match': '\\b(func|var|const|type|struct|interface|case|default)\\b'
+    'match': '\\b(chan|map|func|var|const|type|struct|interface|case|default)\\b'
   }
   {
     'name': 'keyword.directive.go'
-    'match': '\\b(package|import)\\b'
+    'match': '\\bimport\\b'
+  }
+  {
+    'match': '\\b(package)\\b\\s+([\\w_][\\w\\d_]*)?'
```

Comment:
Just noticed the `?` at the end of the regexp and I understand why you added it, but wonder if it's the best way...

Wouldn't it be better to revert this so `package` is matched together with `import` again (like it was before) and then add a regexp like this: `'match': '\\bpackage\\b\\s+([\\w_][\\w\\d_]*)'`

This way `package` will always be scoped correctly, and independent from that word, the package name will also be scoped correctly. It seems this would be a little more robust while having the exact same result.